xbmc/addons/binary-addons/AddonDll.h

```diff
@@ -30,6 +30,18 @@ namespace ADDON
  */
 using ADDON_INSTANCE_HANDLER = const void*;
 
+/*!
+ * @brief Information class for use on addon type managers.
+ *
+ * This to ask via @ref CAddonDll the manager there that work can be performed.
+ * By multiple instances can it be harder to becomes informed about changes.
+ */
```

Comment:
/*!
 * @brief Information class for use on addon type managers.
 *
 * This to query via @ref CAddonDll the manager so that work can be performed.
 * If there are multiple instances it be harder to be informed about any changes.
 */